Fix Generate Dummy Secret Behavior (#3)
argyle-engineering/ksops: README.md

fail-silently: true
files:
- ./secret.enc.yaml
```

## Generate Dummy Secret

> There is some case where our local machine don't have access to the decryptor key, but we still want ksops to just output or keep producing a secret with a dummy value in it. 

Reviewer: ```suggestion
There is a case where our machine does not have an access to the decryptor key (i.e. our CICD), but we still want ksops to keep producing a secret with a placeholder 'secret' value in it.
```
